browser/playlist/playlist_tab_helper.h

```diff
@@ -106,11 +113,14 @@ class PlaylistTabHelper
 
   raw_ptr<PlaylistService> service_;
 
-  GURL target_url;
+  GURL target_url_;
   bool sent_find_media_request_ = false;
 
   bool is_adding_items_ = false;
 
+  bool suspended_media_detection_ = false;
+  std::vector<base::OnceClosure> media_detected_after_suspension_callback_;
```

Comment:
Curious when tab helper can have multiple callbacks?

Reply:
For now it happens only for the test